rdr_service/dao/genomics_dao.py

```diff
@@ -1339,22 +1339,19 @@ def get_wgs_members_files_available(self, sample_list=None):
             return members.all()
 
     @classmethod
-    def get_wgs_pass_date(cls, session, participant_id) -> Optional[datetime]:
-        aw4_job_run = session.query(
-            GenomicJobRun.startTime
+    def get_wgs_pass_date(cls, session, biobank_id) -> Optional[datetime]:
+        aw4_record = session.query(
+            GenomicAW4Raw.created
         ).select_from(
-            GenomicSetMember
-        ).join(
-            GenomicJobRun,
-            GenomicJobRun.id == GenomicSetMember.aw4ManifestJobRunID
+            GenomicAW4Raw
         ).filter(
-            GenomicSetMember.participantId == participant_id,
-            GenomicSetMember.genomeType == config.GENOME_TYPE_WGS,
-            GenomicSetMember.qcStatus == GenomicQcStatus.PASS
-        ).first()
+            GenomicAW4Raw.biobank_id == f'{get_biobank_id_prefix()}{biobank_id}',
+            GenomicAW4Raw.genome_type == config.GENOME_TYPE_WGS,
+            GenomicAW4Raw.qc_status == "PASS"
```

Comment:
Since this string can change via the GCs, should we use `.ilike('pass')` for the filter?

Reply:
Updated.